docs/rundown.rst

```diff
@@ -97,9 +175,11 @@ it. These steps both happen in a single ``ansible-sign`` command.
    :caption: Generating a checksum manifest file and signing it
 
    $ ansible-sign project gpg-sign .
-   GPG signing successful!
-   Checksum manifest: ./.ansible-sign/sha256sum.txt
-   GPG summary: signature created
+[OK   ] GPG signing successful!
+[NOTE ] Checksum manifest: ./.ansible-sign/sha256sum.txt
+[NOTE ] GPG summary: signature created
```

Comment:
These can't be dedented because they won't be in the code bock anymore, rst is indentation dependent.